ArduPlane/quadplane.cpp

```diff
@@ -2488,6 +2488,18 @@ void QuadPlane::vtol_position_controller(void)
 
             // switch to vfwd for throttle control
             vel_forward.integrator = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
+
+            // adjust the initial forward throttle based on our desired and actual closing speed
+            // this allows for significant initial forward throttle
+            // when we have a strong headwind, but low throttle in the usual case where
+            // we want to slow down ready for POSITION2
+            vel_forward.integrator = linear_interpolate(0, vel_forward.integrator,
+                                                        closing_speed,
+                                                        1.2*desired_closing_speed, 0.5*desired_closing_speed);
```

Comment:
```suggestion
            vel_forward.integrator = linear_interpolate(0, vel_forward.integrator,
                                                        closing_speed,
                                                        0.5*desired_closing_speed, 1.2*desired_closing_speed);
```
This is a better match for the usual pattern. The `linear_interpolate` call swaps them in anycase so the result should be the same.